Sketch.java

```diff
@@ -23,14 +26,104 @@ public void setup() {
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
```

Comment:
missing [level 3 requirement](https://github.com/SACHSTech/ProcessingTask6-CreatingMethods#level-3), "draw a composite object at **various locations** specified by the method parameters."